app/app.py

```diff
@@ -15,6 +15,9 @@ def coconut():
     Handles Coconut code submitted by users.
     """
     code = request.form['code']
+    # Get optional compile arguments
+    # Examples: http://coconut.readthedocs.io/en/master/DOCS.html#usage
+    compile_args = request.form.get('args', None)
```

Comment:
The request payload is most likely going to be typed with a default of `''` (empty string) instead of a null value.

Reply:
Done